src/modules/rlm_attr_filter/rlm_attr_filter.c

```diff
@@ -133,11 +133,8 @@ static int attr_filter_getfile(TALLOC_CTX *ctx, rlm_attr_filter_t *inst, char co
 		     * then bitch about it, giving a good warning message.
 		     */
 		     if (fr_dict_attr_is_top_level(vp->da) && (vp->da->attr > 1000)) {
-			 char *str;
-			 fr_value_box_aprint(NULL, &str, entry->name, NULL);
-			 WARN("[%s]:%d Check item \"%s\"\n\tfound in filter list for realm \"%s\".\n",
-			      filename, entry->lineno, vp->da->name, str);
-			 talloc_free(str);
+			 WARN("[%s]:%d Check item \"%s\"\n\tfound in filter list for realm \"%pV\".\n",
```

Comment:
If you have time after, could you fix up these log messages.  Embedded newlines won't be printed.  If the log message should be displayed on multiple lines, it needs multiple calls to the logging function.  Trailing newlines aren't needed either.